sdks/java/extensions/sql/build.gradle

```diff
@@ -24,7 +24,7 @@ plugins {
   id 'ca.coglinc.javacc'
 }
 applyJavaNature(
-  suppressUnusedVariable: true,
+  suppressUnusedVariable: false,
```

Comment:
nit: false is the default, so we could just drop this lines instead.
```suggestion
```

Reply:
Done!